ArticleTemplates/assets/scss/garnett-layout/_article--shared.scss

```diff
@@ -41,14 +41,6 @@
         // Interactives within article body
         .element-interactive {
             width: auto;
-
-            @include mq(col2, col3) {
-                width: calc(#{(100% / (100% - cols($base-3, 3))) * 100%} - #{base-px(1)});
-            }
-
-            @include mq($from: col3) {
-                width: calc(#{(100% / (100% - cols($base-4, 4))) * 100%} - #{base-px(1)});
-            }
         }
```

Comment:
@duarte these style rules were copied over from pre-garnett world. They look like they're trying to do something very clever, I'm not really sure what to be honest, they however caused this bug so I've removed them. Let me know if you think there's an alternative.

Reply:
I think this was broken by a garnett PR I did where I changed the text column width on tablet. It's possible that if we update the cols() parameters to match the ones on lines 10 and 14 on _article--shared.scss, it would restore the full-width functionality.

https://github.com/guardian/mobile-apps-article-templates/blob/1d43f00676a64dcb43e9ab7249de39d4aec26781/ArticleTemplates/assets/scss/garnett-layout/_article--shared.scss#L10